pkg/internal/constants.go

```diff
@@ -1,7 +1,8 @@
 package internal
 
 const (
-	ConfigNamespace    = "openshift-config"
-	InstallerConfigMap = "openshift-install"
-	ManifestsConfigMap = "openshift-install-manifests"
+	ConfigNamespace         = "openshift-config"
+	InstallerConfigMap      = "openshift-install"
+	ManifestsConfigMap      = "openshift-install-manifests"
+	ClusterProfileConfigMap = "cluster-profile"
```

Comment:
There are a few comments in the PR about if/how to use a ConfigMap to store this config (which was [out-of-scope][1] in the enhancement), starting [here][2].  Can we move them to this thread so they don't get interleaved with other PR-scoped comments?  I feel like managing this config may deserve its own enhancement :p.  Is this intended to be a install-time config?  What happens if admins twiddle the value post-install?  What happens if they set an invalid value (seems like the CVO blindly accepting an invalid profile would effectively cripple manifest application)?  Should we be reporting the configured profile through metrics/telemetry?  My initial knee-jerk is that this should be a ClusterVersion `status` property, to dodge some of these mutable-location wrinkles, but we're unlikely to be able to alter ClusterVersion without a formal enhancement.  Thoughts?

[1]: https://github.com/openshift/enhancements/pull/200#discussion_r394654607
[2]: https://github.com/openshift/cluster-version-operator/pull/404#issuecomment-662516200

Reply:
For CRC, it is definitely an install-time config and we don't want the admin to modify it.
In this PR, the ClusterVersion object is key and the ConfigMap with the profile must be created before (as the CV is a lock for the sync loop). The admin have to insert the correct manifest between the `openshift-install create manifests` and `openshift-install create cluster` phases.  
I must admit it would cleaner if we can have a profile property in the CV like CVO overrides.
Is this first version acceptable still? WDYT?

Reply:
@guillaumerose Why do you think CRC need it as install-time config? To me it should work for CRC even if you provide the ConfigMap post the first installation.

Reply:
It should work post-install but it will take extra time (10-20min?) to install unnecessary components. It can also have weird side effects. It would be great to guaranty that a profile is not dependent of the default profile.

Reply:
Any change to API also impact to multiple components (Installer, CVO) should be an enhancement https://github.com/openshift/enhancements#is-my-thing-an-enhancement

Reply:
To move this PR forward, I propose to:
* remove the use of the ConfigMap,
* only use the env. variable (as stated in the enhancement).
* open a PR in the installer to add the env variable.

WDYT?

Reply:
You'd need to adjust the `render` logic here and also pur default CVO Deployment templating to propagate the env var between updates.  But sounds plausible to me.

Reply:
I updated the PR with only the env variable and the flag. It's now clearer although the flag `--cluster-profile` is a little bit invasive. Let me know what you think.